bin/node/src/commands/block_producer.rs

```diff
@@ -28,8 +25,12 @@ pub enum BlockProducerCommand {
         #[command(flatten)]
         block_producer: BlockProducerConfig,
 
-        #[command(flatten)]
-        telemetry: TelemetryConfig,
+        /// Enables the exporting of traces for OpenTelemetry.
+        ///
+        /// This can be further configured using environment variables as defined in the official
+        /// OpenTelemetry documentation. See our operator manual for further details.
+        #[arg(long = "enable-otel", default_value_t = false, env = ENV_ENABLE_OTEL, value_name = "BOOL")]
+        open_telemetry: bool,
```

Comment:
nit: should we rename this into `enable_otel` to be consistent with the arg naming?